appendix-keyboard.tex

```diff
@@ -23,8 +23,8 @@ \section{Hardware Accelerated Keyboard Scanning}
 
 The hardware accelerated keyboard scanner is very simple to use: First, make sure
 that you have the MEGA65 IO context activated, then read memory location \$D610
-(decimal 54800). If the contents are zero, no key has been pressed.  Otherwise the
-value with be the ASCII code of the most recent key or key-combination that has
+(decimal 54800). If the contents is zero, no key has been pressed.  Otherwise the

```

Comment:
Both "is" and "are" are a bit awkward here.  If we instead say "If the register contains zero" to avoid the grammatical awkwardness.

Reply:
Fixed 7acd9befbc90f2a14fd6b67db5cdb99ae5346d29